go/pfqueueclient/pfqueueclient.go

```diff
@@ -53,7 +53,7 @@ func (c *PfQueueClient) SubmitWithExpiration(ctx context.Context, queue, task_ty
 	}
 
 	redisClient.PipeAppend("MULTI")
-	redisClient.PipeAppend("HMSET", id, "expire", expire_in, "data", data)
+	redisClient.PipeAppend("HMSET", id, "expire", expire_in, "data", data, "tenant_id", tenant_id)
```

Comment:
Can you confirm that there is existing magic to set the tenant ID in the DAL and config in pfqueue?
I don't see any changes related to that so I just want to be sure its already there.

Reply:
It has been there for a long time look at the commit a998fff4.